dockerfiles/ubuntu/obam-uk/Dockerfile

```diff
@@ -0,0 +1,104 @@
+This Docker container comprises of The WSO2 Open Banking Solution, with its latest GA release \n\
+which is under The WSO2 Commercial License 1.0. \n\
+Read more about The WSO2 Commercial License, Version 1.0 here @ http://wso2.com/licenses \n"
+
+# configuration properties for deployment.toml file
+# server hostnames
+ARG IS_HOSTNAME=obiam
+ARG APIM_HOSTNAME=obam
+ARG BI_HOSTNAME=localhost
+# databases related properties
+ARG DB_TYPE=mysql
+ARG DB_USER=root
+ARG DB_PASS=root
+ARG DB_HOST=mysql
+ARG DB_DRIVER=com.mysql.jdbc.Driver
+ARG DB_APIMGT=openbank_apimgtdb
+ARG DB_USER_STORE=openbank_userdb
+ARG DB_AM_CONFIG=openbank_am_configdb
+ARG DB_GOV=openbank_govdb
+ARG DB_IS_CONFIG=openbank_iskm_configdb
+ARG DB_OPEN_BANKING_STORE=openbank_openbankingdb
+ARG DB_OB_REPORTING=openbank_ob_reporting_statsdb
+ARG DB_OB_REPORTING_SUMMARIZED=openbank_ob_reporting_summarizeddb
+
+# change user to root for elevated permissions to override inherited non-root user
+USER root
+
+# add Open Banking artifacts to base product
+RUN \
+    wget -O ${WSO2_OB_TOOLKIT}.zip "${WSO2_OB_TOOLKIT_DIST_URL}" \
+    && unzip -d ${WSO2_SERVER_HOME}/ ${WSO2_OB_TOOLKIT}.zip \
+    && rm -f ${WSO2_OB_TOOLKIT}.zip
+
+# set configurations
+# read deployment.toml file
+ARG DEPLOYMENT_TOML_FILE=${WSO2_OB_TOOLKIT_DIR}/repository/resources/deployment.toml
+RUN \
+    cp ${WSO2_OB_TOOLKIT_DIR}/repository/resources/${WSO2_SERVER}-deployment-uk.toml ${DEPLOYMENT_TOML_FILE} \
+# configure hostnames
+    && sed -i -e 's|IS_HOSTNAME|'${IS_HOSTNAME}'|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|APIM_HOSTNAME|'${APIM_HOSTNAME}'|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|BI_HOSTNAME|'${BI_HOSTNAME}'|g' ${DEPLOYMENT_TOML_FILE} \
+# configure datasources
+    && sed -i -e 's|DB_APIMGT_URL|jdbc:mysql://'${DB_HOST}':3306/'${DB_APIMGT}'?autoReconnect=true\&amp;useSSL=false|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|DB_AM_CONFIG_URL|jdbc:mysql://'${DB_HOST}':3306/'${DB_AM_CONFIG}'?autoReconnect=true\&amp;useSSL=false|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|DB_GOV_URL|jdbc:mysql://'${DB_HOST}':3306/'${DB_GOV}'?autoReconnect=true\&amp;useSSL=false|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|DB_USER_STORE_URL|jdbc:mysql://'${DB_HOST}':3306/'${DB_USER_STORE}'?autoReconnect=true\&amp;useSSL=false|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|DB_USER|'${DB_USER}'|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|DB_PASS|'${DB_PASS}'|g' ${DEPLOYMENT_TOML_FILE} \
+    && sed -i -e 's|DB_DRIVER|'${DB_DRIVER}'|g' ${DEPLOYMENT_TOML_FILE} \
+# copy deployment.toml file to repository/conf
+    && cp ${DEPLOYMENT_TOML_FILE} ${WSO2_SERVER_HOME}/repository/conf/ \
+    && rm ${DEPLOYMENT_TOML_FILE} 
+
+# copy the WSO2 OB APIM UK Toolkit artifacts on top of WSO2 APIM base product
+# Remove default backend war
+RUN \
+    rm ${WSO2_SERVER_HOME}/repository/deployment/server/webapps/api#openbanking#backend.war\
+# copy open banking artifacts
```

Comment:
lets change the comment to copy toolkit artifacts

Reply:
done